packages/legend-extension-format-graphql/package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "@finos/legend-extension-format-graphql",
+  "version": "1.0.0",
```

Changeset will automatically bump this upon release, this should be set to `0.0.0` for now

```suggestion
  "version": "0.0.0",
```

---

done